pkg/apis/workflow/v1alpha1/cron_workflow_types.go

```diff
@@ -69,8 +78,21 @@ type CronWorkflowStatus struct {
 	LastScheduledTime *metav1.Time `json:"lastScheduledTime" protobuf:"bytes,2,opt,name=lastScheduledTime"`
 	// Conditions is a list of conditions the CronWorkflow may have
 	Conditions Conditions `json:"conditions" protobuf:"bytes,3,rep,name=conditions"`
+	// Succeeded is a counter of how many times the child workflows had success
```

Comment:
```suggestion
	// v3.6 and after: Succeeded counts how many times the child workflows succeeded
```
a bit simpler plus a version specifier